history/cultures/alteraci.txt

```diff
@@ -1,51 +1,75 @@
 1.1.1 = {
```

Comment:
Blackwater culture (from the goblin group) isn't on the goblin tech level.

Reply:
should pirate and wastewander be on human level then?  I would assume all pirate/bandit cultures are a little behind.

Reply:
> should pirate and wastewander be on human level then? I would assume all pirate/bandit cultures are a little behind.

Southern and Wastewander should be on the Kul Tiran level.

Reply:
We had these cultures a lot lower on the tech tier spreadsheet before, lets redo the sheet and I will implement